packages/backend/src/clients/dbtCloud/typeTransformers.ts

```diff
@@ -5,11 +5,22 @@ import {
     DbtMetricType,
     DbtTimeGranularity,
     DbtWhereOperator,
+    isSemanticLayerBaseOperator,
+    isSemanticLayerExactTimeFilter,
+    isSemanticLayerRelativeTimeFilter,
+    isSemanticLayerStringFilter,
+    isSemanticLayerTimeFilter,
     SemanticLayerFieldType,
     SemanticLayerFilter,
-    SemanticLayerStringFilterOperator,
+    SemanticLayerFilterBaseOperator,
+    SemanticLayerFilterRelativeTimeValue,
     SemanticLayerTimeGranularity,
+    type SemanticLayerExactTimeFilter,
+    type SemanticLayerRelativeTimeFilter,
+    type SemanticLayerStringFilter,
+    type SemanticLayerTimeFilter,
 } from '@lightdash/common';
+import moment from 'moment';
```

Comment:
moment is not maintained anymore and we've been trying to get rid of it. im happy to pitch `date-fns` over `day.js` (which we use in the FE but I prefer date-fns)